_datascience/cbioportal.md

```diff
@@ -1 +1,34 @@
+---
+title: cBioPortal
+primary_reviewers: vortexing, abbywall
+---
+
+[**cBioPortal**](https://www.cbioportal.org/) is an open-access, open-source resource for interactive exploration of multidimensional cancer genomics data sets developed by Memorial Sloan Kettering (MSK). The Data Science Lab (DaSL) in conjunction with the Scientific Computing Group (SciComp) at Fred Hutch have integrated a Fred Hutch instance of cBioPortal for research use within the Fred Hutch. 
+
+The Fred Hutch deployment of cBioPortal allows users to:
+
+- 
+- 
+- 
+
+## Getting Started
+
```

Comment:
Visit the cBioPortal website to learn more about[ what cBioPortal](https://docs.cbioportal.org/) is and how it can facilitate your research. You can get [in-depth](https://docs.cbioportal.org/file-formats/#introduction) information on how to prepare your study for upload into the public instance and a lot of information about how to use cBioPortal etc.

Reply:
keeping this bullet points - @sitapriyamoorthi you should review as is now